00001-enarx-sprint-framework/README.md

```diff
@@ -0,0 +1,612 @@
+- Development team: people working on individual components of the project. Not
+  necessarily developers; may also encapsulate docs writers, outreach, or other
+  roles.
+
+This proposal will focus most broadly on the responsibilities of individual
+development team members as well as project leadership from the above
+definitions.
+
+## Common Concepts
+
+A few things will be referenced throughout the proposal; they will be defined
+here.
+
+### Triage
+
+Triage is where the team will categorize newly-filed issues and pull requests,
+and where product owners will be able to express needed urgency over those
+items. It's an important process that is used in several places throughout this
+document, so it's being introduced as its own concept.
+
+The process is illustrated in the following flowchart:
+
+![Triage Diagram](./triage.svg)
+
+Issues and PRs are deliberately separated, since they represent different units
+of work and should be handled as such.
+
+Firstly, incoming issues and PRs are labeled according to the category or
+subdomain of the project. For example, they may be labeled as `sgx`, `sev` or
+`infrastructure`. This categorization is undertaken by the team as a whole.
+
+From here, the process differs:
+
+- Issues represent descriptions of work that may need to be done. These can
+  range from a low-priority polish task to an urgent security flaw. Recognizing
+  this, the next step in the process is a decision by the stakeholders:
+  **is this urgent?** If so, it should be assigned to a team member and
+  promoted to the current sprint. Doing so recognizes its importance and
+  makes it an immediate priority. If it's not considered urgent, it will be
+  dropped in the project's backlog for later consideration.
+
+- PRs represent work that has already been done and needs to be reviewed before
+  being merged into the main codebase. As such, they will almost always be
+  attached to an issue in the current sprint, and should be treated as an
+  extension of that issue. The sole purpose of triaging a new PR is to ensure it
+  has reviewers requested and assigned. Once that's done, it gets put in the
+  current sprint.
+
+### Project Board
+
+A common concept in many project workflows, the **project board** is also how the
+Enarx project organizes its work.
+
+Currently, the project uses multiple area-specific boards. This proposal
+abandons that idea and instead uses a single, unified project board that
+contains the entire project backlog and all current work.
+
+The board will be organized with the following columns:
+
+- backlog: collection of issues not currently being worked on
+- nominated: issues nominated by team members for the next sprint (planning)
+- accepted: issues the team should work on next sprint (planning)
+- assigned: issues actively being worked on by an assigned team member
+- under review: all submitted PRs go here while awaiting review
+- done: all issues and PRs go here when done or merged, respectively
```

Comment:
We need one additional board: `triage`. The bot should put all open issues that are not in the project board into the `triage` column.